### PR TITLE
Community page redesign update

### PR DESCRIPTION
**Pull request by dbuzhorkaa**

### CODE REVIEW

**Comment by dbuzhorkaa** [comment]
Fixed review comments. @bdmogal please recheck

**Comment by bdmogal** [comment]
Some of the text is not correct @dbuzhorkaa. Could you please update the top three sections from the doc again?

Ask a question
You get in touch with the community using any of the following channels:
[Mailing List] [Slack] [Stack Overflow]

Report a bug
To report bugs, please use the Jira Issue Tracker. We recommend you to post your issues to the mailing list, and report a bug if it is confirmed as an issue there.
[Jira Link to https://issues.cask.co/] 

Join the BDAM group
The Big Data Application Meetup is a community focused on solving real world problems using Big Data technologies. 
[Meetup link - https://www.meetup.com/BigDataApps/]

**Comment by dbuzhorkaa** [comment]
@bdmogal done. Please recheck

File: data/en/community.json [hunk 1]
```diff
@@ -33,18 +33,18 @@
         },
         {
           "title": "Report a bug",
-          "body": "Tracker. We recommend you to post your issues to the mailing list, and report a bug if it is confirmed as an issue there.",
+          "body": "To report bugs, please use the Jira Issue Tracker. We recommend you to post your issues to the mailing list, and report a bug if it is confirmed as an issue there.",
```
Comment (bdmogal) [inline data/en/community.json]: lower case issue tracker.

File: data/en/community.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
       "items": [
         {
           "title": "Ask a question",
-          "body": "You get in touch with the community using any of the following links:",
+          "body": " You get in touch with the community using any of the following channels:",
```
Comment (bdmogal) [inline data/en/community.json]: remove unnecessary space in the beginning?

File: data/en/community.json (hunk 1 above)
Comment (dbuzhorkaa) [reply data/en/community.json]: done

File: data/en/community.json (hunk 2 above)
Comment (dbuzhorkaa) [reply data/en/community.json]: done

**Pull request merged**